theano/tensor/tests/test_opt.py

```diff
@@ -4016,17 +4017,15 @@ def test_local_useless_tile(self):
         if theano.config.mode == "FAST_COMPILE":
             mode = "FAST_RUN"
         for var, data in [(v, [1, 2, 3]), (m, [[1, 2], [3, 4]])]:
-            # Currently, only a repeat patter == ndim is supported.
-            for ndim in [var.ndim]:  # range(1, var.ndim):
-                f = theano.function([var], Tile(ndim)(var, (1,)*ndim), mode=mode)
-                topo = f.maker.fgraph.toposort()
-                assert len(topo) == 1
-                assert isinstance(topo[0].op, compile.DeepCopyOp)
-                f(data)
-                
-                # Check that stacktrace is copied over
-                self.assertTrue(hasattr(f.outputs[0].variable.tag, 'trace'))
-                self.assertTrue(len(f.outputs[0].variable.tag.trace)>0)
+            f_tile_op = theano.function([var], Tile(var.ndim)(var, (1,) * var.ndim), mode=mode)
+            f_tile_th = theano.function([var], tile(var, (1,) * var.ndim), mode=mode)
+            topo = f_tile_th.maker.fgraph.toposort()
+            assert len(topo) == 1
```

Comment:
The point of that test is that we want, ultimately, to add the optimizations so that it passes.
We will take care of the for loop later.